scripts/daily_sentinel_operational_check.py

```
@@ -0,0 +1,506 @@
+    context.check_hostname = False
+    context.verify_mode = ssl.CERT_NONE
+    return context
+
+
+def _open_dashboard(
+    url: str,
+    method: str,
+    timeout_seconds: float,
+    context: ssl.SSLContext | None,
+) -> int:
+    request = urllib.request.Request(url, method=method)
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)